build.gradle

```diff
@@ -78,10 +78,34 @@ subprojects {
           artifact sourcesJar
 
           pom {
+            name = 'Google Cloud Common Protos for Java'
             packaging = 'jar'
             artifactId = project.name
-            description = 'Google Common Protos for Java'
-
+            description = 'Google Cloud Common Protos for Java'
+            url = 'https://github.com/googleapis/common-protos-java'
+
+            scm {
+              url = 'https://github.com/googleapis/common-protos-java'
+              connection = 'scm:git:https://github.com/googleapis/common-protos-java.git'
+            }
+
+            licenses {
+              license {
+                name = 'Apache'
+                url = 'https://github.com/googleapis/common-protos-java/blob/master/LICENSE'
+              }
+            }
+
+            developers {
+              developer {
+                id = 'GoogleAPIs'
+                name = 'GoogleAPIs'
+                email = 'googleapis@googlegroups.com'
+                url = 'https://github.com/googleapis/common-protos-java'
+                organization = 'Google, Inc.'
```

Comment:
Google LLC. Probably should be fixed for gax-java script as well.

Reply:
Done.